renku/cli/login.py

```diff
@@ -35,66 +35,46 @@
 
     The warning will be shown when an unsecure protocol is used.
 
-Non-interactive login
-~~~~~~~~~~~~~~~~~~~~~
-
-In some environments, you might need to run the ``renku login`` command
-non-interactively. Using the ``--password-stdin`` flag, you can provide a
-password through ``STDIN``, which also prevents the password from ending up in
-the shell's history or log-files.
-
-The following example reads a password from a file, and passes it to the
-``renku login`` command using ``STDIN``:
-
-.. code-block:: console
-
-    $ cat ~/my_secret.txt | renku login --username demo --password-stdin
-
 """
 
+import uuid
+import webbrowser
+
 import click
+import requests
 
 from ._client import from_config
 from ._config import config_path, with_config
-from ._options import argument_endpoint, default_endpoint, password_prompt
+from ._options import argument_endpoint, default_endpoint
 
 
 @click.command()
 @click.argument('endpoint', required=False, callback=default_endpoint)
-@click.option(
-    '--url',
-    default='{endpoint}/auth/realms/Renku/protocol/openid-connect/token'
-)
-@click.option('--client-id', default='demo-client')
-@click.option('--username', prompt=True)
-@click.option('--password', callback=password_prompt)
-@click.option('--password-stdin', is_flag=True)
+@click.option('--url', default='{endpoint}/auth')
 @click.option('--default', is_flag=True)
 @with_config
 @click.pass_context
-def login(
-    ctx, config, endpoint, url, client_id, username, password, password_stdin,
-    default
-):
+def login(ctx, config, endpoint, url, client_id, default):
```

Comment:
I get an exception here about a non-existing positional argument (client_id). It is not used in the code though so simply removing the argument fixed the issue for me.